core/frontend/src/quantity-formatting/BasicUnitsProvider.ts

```diff
@@ -186,6 +186,7 @@ const UNIT_DATA: UnitDefinition[] = [
   { name: "Units.SQ_FT", phenomenon: "Units.AREA", system: "Units.USCUSTOM", conversion: { numerator: 1.0, denominator: .09290304, offset: 0.0 }, displayLabel: "ft²" },
   { name: "Units.SQ_US_SURVEY_FT", phenomenon: "Units.AREA", system: "Units.USCUSTOM", conversion: { numerator: 15499969.0, denominator: 1440000, offset: 0.0 }, displayLabel: "ft² (US Survey)" },
   { name: "Units.SQ_M", phenomenon: "Units.AREA", system: "Units.SI", conversion: { numerator: 1.0, denominator: 1.0, offset: 0.0 }, displayLabel: "m²" },
+  { name: "Units.SQ_KM", phenomenon: "Units.AREA", system: "Units.SI", conversion: { numerator: 1.0, denominator: 1000000.0, offset: 0.0 }, displayLabel: "km²" },
```

Comment:
please add a test that verifies that the result of the conversion is as expected.  It seems like most existing tests using the BasicUnitsProvider are tied up with the QuantityFormatter test but that is OK. You can add a test there.